deepchecks/utils/distribution/drift.py

```diff
@@ -70,13 +71,31 @@ def cramers_v(dist1: Union[np.ndarray, pd.Series], dist2: Union[np.ndarray, pd.S
         array of numerical values.
     dist2 : Union[np.ndarray, pd.Series]
         array of numerical values to compare dist1 to.
+    min_category_size_ratio: float, default 0.01
+        minimum size ration for categories. Categories size ratio than this number are binned into an "Other" category.
+    max_num_categories: int, default: None
+        max number of allowed categories. If there are more categories than this number, categories are ordered by
+        magnitude and all the smaller categories are binned into an "Other" category.
+        If max_num_categories=None, there is no limit.
+        > Note that if this parameter is used, the ordering of categories (and by extension, the decision which
+        categories are kept by name and which are binned to the "Other" category) is done by default according to the
+        values of dist1, which is treated as the "expected" distribution. This behavior can be changed by using the
+        sort_by parameter.
+    sort_by: str, default: 'difference'
+        Specify how categories should be sorted, affecting which categories will get into the "Other" category.
+        Possible values:
+        - 'dist1': Sort by the largest dist1 categories.
+        - 'dist2': Sort by the largest dist2 categories.
+        - 'difference': Sort by the largest difference between categories.
+        > Note that this parameter has no effect if max_num_categories = None or there are not enough unique categories.
     Returns
     -------
     float
         the bias-corrected Cramer's V value of the 2 distributions.
 
     """
-    dist1_counts, dist2_counts, _ = preprocess_2_cat_cols_to_same_bins(dist1=dist1, dist2=dist2)
+    dist1_counts, dist2_counts, _ = preprocess_2_cat_cols_to_same_bins(dist1, dist2, min_category_size_ratio,
+                                                                       max_num_categories, sort_by)
```

Comment:
instead of inserting these parameters into cramers_v, maybe take them out and put them in calc_drift? Like we do in PSI. Seems simpler and makes more sense module-wise